components/mailbox/src/init.spec.js

```diff
@@ -607,6 +607,116 @@ describe("Mailbox Pagination: threads not intercepted", () => {
+      "https://web-components.nylas.com/middleware/manifest",
+      {
+        fixture: "mailbox/manifest.json",
+      },
+    );
+    cy.intercept("GET", "https://web-components.nylas.com/middleware/account", {
+      fixture: "mailbox/account.json",
+    });
+    cy.intercept("GET", "https://web-components.nylas.com/middleware/labels", {
+      fixture: "mailbox/labels.json",
+    });
+
+    cy.intercept(
+      "GET",
+      "https://web-components.nylas.com/middleware/threads?view=expanded&not_in=trash&limit=13&offset=0&in=inbox",
+      {
+        fixture: "mailbox/threads/threadWithLastMessageFromSelf",
+      },
+    ).as("threadWithLastMessagefromSelf");
+    cy.intercept(
+      "GET",
+      "https://web-components.nylas.com/middleware/messages/last-message-from-self",
+      {
+        fixture: "mailbox/messages/idWithLastMessageFromSelf",
+      },
+    );
+
+    cy.visit("/components/mailbox/src/cypress.html");
+
+    cy.get("nylas-mailbox").should("exist").as("mailbox");
+    cy.get("@mailbox").find("nylas-email").as("email");
+  });
+
+  it(`On click of reply, shows correct to field`, () => {
+    cy.wait("@threadWithLastMessagefromSelf");
+    cy.get("@mailbox").invoke("prop", "show_reply", true);
+    cy.get("@mailbox").invoke("prop", "show_reply_all", true);
+    cy.get("@mailbox").find(".email-row.condensed").click();
+
+    cy.get("@email").find(".reply").should("exist");
+    cy.get("@email").find(".reply").click();
+    cy.get("nylas-composer").should("exist").as("composer");
+    cy.get("nylas-composer")
+      .find("[data-cy=to-field] .contact-item__name")
+      .should("have.text", "Ricky Ropora <rickyropora@gmail.com>");
+  });
+});
+
+describe.only("Mailbox: updating cc fields correctly for reply and reply-all", () => {
+  beforeEach(() => {
+    cy.intercept(
+      "GET",
+      "https://web-components.nylas.com/middleware/manifest",
+      {
+        fixture: "mailbox/manifest.json",
+      },
+    );
+    cy.intercept("GET", "https://web-components.nylas.com/middleware/account", {
+      fixture: "mailbox/account.json",
+    });
+    cy.intercept("GET", "https://web-components.nylas.com/middleware/labels", {
+      fixture: "mailbox/labels.json",
+    });
+
+    cy.intercept(
+      "GET",
+      "https://web-components.nylas.com/middleware/threads?view=expanded&not_in=trash&limit=13&offset=0&in=inbox",
+      {
+        fixture: "mailbox/threads/threadWithMultipleToParticipants",
+      },
+    ).as("threadWithMultipleToParticipants");
+    cy.intercept(
+      "GET",
+      "https://web-components.nylas.com/middleware/messages/message-with-multiple-to-paticipants",
+      {
+        fixture: "mailbox/messages/idWithMultipleToParticipants",
+      },
+    );
+
+    cy.visit("/components/mailbox/src/cypress.html");
+
+    cy.get("nylas-mailbox").should("exist").as("mailbox");
+    cy.get("@mailbox").find("nylas-email").as("email");
+  });
+
+  it(`On click of reply-all, shows correct cc fields`, () => {
```

Comment:
We'd also add composer to here only to keep it isolated as every other test doesn't need composer.
```suggestion
  it(`On click of reply-all, shows correct cc fields`, () => {
  
  cy.addComponent("nylas-composer", {
        show_header: true,
        show_minimize_button: false,
        show_from: false,
        show_bcc: false,
        show_cc: true,
        reset_after_send: true,
        reset_after_close: true,
    });
    
    cy.get('nylas-composer').as('composer').then(element => {
       const composer = element[0];
       
       cy.get('@mailbox').then(el => {
          const mailbox = el[0];
       
            mailbox.addEventListener("replyAllClicked", (event) => {
                composer.removeAttribute("hidden");
                composer.value = event.detail.value;
                composer.focus_body_onload = event.detail.focus_body_onload;
                if (Object.keys(event.detail.message).length) {
                  composer.message_with_body = event.detail.message;
                }
                composer.open();
              });
        });
        composer.close();
    });
```